server/common/app_config.py

```diff
@@ -442,6 +447,19 @@ def handle_diffexp(self, context):
                         f"running differential expression may take longer or fail."
                     )
 
+        max_workers = self.diffexp__alg_tiledb__max_workers
+        if type(max_workers) == str:
+            cpu_count = os.cpu_count()
+            try:
+                max_workers = eval(max_workers, {"cpu_count": cpu_count})
```

Comment:
having an eval in the code makes me very squeamish from a reliability and infosec standpoint....   Is there a reason we need general purpose scripting here, rather than a much simpler set of params (eg, "max_threads" and "cpu_multiplier"?)

Reply:
Yeah good point.  Not sure what I was thinking